Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN apt-get update && apt-get install -y \
 ENV AFL_HARDEN 1
 WORKDIR $AFL_DIR
 RUN wget http://lcamtuf.coredump.cx/afl/releases/afl-1.94b.tgz
```

Comment:
This needs to be updated as well :-) Might want to factor AFL version out into an env vars